src/libxtp/davidsonsolver.cc

```diff
@@ -271,18 +275,17 @@ DavidsonSolver::ProjectedSpace DavidsonSolver::initProjectedSpace(
   return proj;
 }
 
-long DavidsonSolver::extendProjection(DavidsonSolver::RitzEigenPair &rep,
-                                      DavidsonSolver::ProjectedSpace &proj) {
+Index DavidsonSolver::extendProjection(const DavidsonSolver::RitzEigenPair &rep,
+                                       DavidsonSolver::ProjectedSpace &proj) {
 
   Index nupdate = 0;
   for (Index j = 0; j < proj.size_update; j++) {
 
-    // skip the root that have already converged
-    if (this->_matrix_type == MATRIX_TYPE::SYMM) {
```

Comment:
I know I introduced that for "a" reason. It might speed up convergence ... would be nice to test again with it

Reply:
do you want to do it or should I? I have no idea what the reason was. :)